src/bentoml/_internal/bento/bento.py

```diff
@@ -88,29 +91,14 @@ def get_default_svc_readme(svc: Service, svc_version: str | None = None) -> str:
         else:
             svc_version = "None"
 
-    doc = f"""\
-# {svc.name}:{svc_version}
-
-[![pypi_status](https://img.shields.io/badge/BentoML-{bentoml_version}-informational)](https://pypi.org/project/BentoML)
-[![documentation_status](https://readthedocs.org/projects/bentoml/badge/?version=latest)](https://docs.bentoml.org/)
-[![join_slack](https://badgen.net/badge/Join/BentoML%20Slack/cyan?icon=slack)](https://l.bentoml.com/join-slack-swagger)
-[![BentoML GitHub Repo](https://img.shields.io/github/stars/bentoml/bentoml?style=social)](https://github.com/bentoml/BentoML)
-[![Twitter Follow](https://img.shields.io/twitter/follow/bentomlai?label=Follow%20BentoML&style=social)](https://twitter.com/bentomlai)
-
-This is a Machine Learning Service created with BentoML."""
-
-    if svc.apis:
-        doc += f"\n{create_inference_api_table(svc)}\n\n"
-
-    doc += """
-
-## Help
+    template = Template(readme)
+    doc = template.render(
+        svc=svc,
+        svc_version=svc_version,
+        create_inference_api_table=create_inference_api_table,
+        bentoml_version=bentoml_version,
+    )
 
-* [📖 Documentation](https://docs.bentoml.org/en/latest/): Learn how to use BentoML.
-* [💬 Community](https://l.bentoml.com/join-slack-swagger): Join the BentoML Slack community.
-* [🐛 GitHub Issues](https://github.com/bentoml/BentoML/issues): Report bugs and feature requests.
-* Tip: you can also [customize this README](https://docs.bentoml.org/en/latest/concepts/bento.html#description).
-"""
     # TODO: add links to documentation that may help with API client development
```

Comment:
Can you revert this? The default generated readme should be as it. Should user provide a Jinja template under the docs section, you can then use Template to parse it.

It is also better to create a `Environment` object to parse the jinja template.